ui/conf/routes

```diff
@@ -23,6 +23,7 @@ GET        /recipe/publish/:recipeId                                      contro
 GET        /admin                                                         controllers.Application.adminLandingPage
 GET        /admin/recent-activity                                         controllers.Application.recentActivity
 GET        /admin/recent-activity-csv                                     controllers.Application.recentActivityCSV
+GET        /admin/recent-user-activity/:userEmail                         controllers.Application.recentUserActivity(userEmail)
```

Comment:
Related to my point above about choosing the userEmail, you could choose to provide GET and POST handlers on this endpoint. The former might show a form for selecting the user and the latter display the page showing that user's activity (this one being what you have now).